test/release/examples/primers/fileIO.chpl

```diff
@@ -192,7 +192,7 @@ if example == 0 || example == 3 {
 // we can optimize better (using ``mmap``, if you like details).
   {
     var f = open(testfile, iomode.r,
-                 hints=IOHINT_RANDOM|IOHINT_CACHED|IOHINT_PARALLEL);
+                hints=ioHintSet.random | ioHintSet.prefetch | ioHintSet.direct(QIO_HINT_PARALLEL));
```

Comment:
Since this is a primer, we should only use publicly supported symbols.  That suggests removing the use of `.direct`.